Editor/Scripts/Server/EditorServerSettings.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+[FilePath("Assets/MeshSyncAssets/EditorServerSettings.yml", FilePathAttribute.Location.ProjectFolder)]
+internal class EditorServerSettings : ScriptableSingleton<EditorServerSettings> {
```

Comment:
Probably better to put this under `Assets/ProjectSettings`

Reply:
Good point, will do